### PR TITLE
add(main): new package alembic

### PR DESCRIPTION
**Pull request by T-Dynamos**
Second step towards #11674

### CODE REVIEW

**Comment by T-Dynamos** [comment]
`LICENSE.txt` exists here https://github.com/alembic/alembic/blob/master/LICENSE.txt
My step 
```
termux_step_install_license() {
	install -Dm600 -t $TERMUX_PREFIX/share/doc/$TERMUX_PKG_NAME \
		$TERMUX_PKG_BUILDER_DIR/LICENSE.txt
}
```
Why fails with 
```
 install: cannot stat '/home/builder/termux-packages/packages/alembic/LICENSE.txt': No such file or directory
```

**Comment by buttaface** [comment]
`TERMUX_PKG_BUILDER_DIR` is where these packaging scripts are, you want `TERMUX_PKG_SRCDIR` instead.

File: packages/alembic/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://alembic.io
+TERMUX_PKG_DESCRIPTION="Alembic is framework for sharing scene data that includes a C++ lib, a file format and client plugin"
+TERMUX_PKG_LICENSE="custom"
```
[the remark this answers is not in the document]
Comment (T-Dynamos) [reply packages/alembic/build.sh:3]: > https://github.com/alembic/alembic/blob/6253269ad35db108ed7ab26a70fb3c4183cb710e/LICENSE.txt#L68

`Boost Software License` is not in https://github.com/termux/termux-packages/blob/master/scripts/lint-packages.sh

So just include BSD ?

[the remark this answers is not in the document]
Comment (buttaface) [reply packages/alembic/build.sh]: [Looks like it is, ie `BSL-1.0`](https://github.com/termux/termux-packages/blob/master/scripts/lint-packages.sh#L19).

**Review by buttaface — APPROVED** [review]
Thanks for submitting this.

**Review by 2096779623 — APPROVED** [review]

**Pull request merged**